.gitignore

```diff
@@ -5,3 +5,7 @@ node_modules/
 npm-debug.log
 yarn.lock
 .eslintcache
+
+# eslint-remote-tester
+.cache-eslint-remote-tester
```

Comment:
Cache is nowadays stored under `node_modules` at `node_modules/eslint-remote-tester/.cache-eslint-remote-tester/`, so no need to specify it in `.gitignore`.

```suggestion
```